CONTRIBUTING.md

```diff
@@ -52,92 +19,9 @@ Please try to report complete, well described, reproducible bugs to us. If you c
-> | [David Brace](https://www.linkedin.com/in/dbrace/ "Head of Product Mgmt & Strategy") | :star: <kbd>**[@orville-wright](https://github.com/orville-wright "Dave Brace")**</kbd> | ![](https://github.com/openebs/community/blob/develop/images/flags/ni_tn/nz.png "I'm a Kiwi from New Zealand. I am based in San Francisco, USA (GMT-7) Timezone") &nbsp; ![](https://github.com/openebs/community/blob/develop/images/flags/de_je/hu.png "I'm also Hungarian. I'm based in San Francisco, USA (GMT-7) Timezone") &nbsp; ![](https://github.com/openebs/community/blob/develop/images/flags/to_zw/us.png "Yes, I'm also an American. I'm based in San Francisco, USA (GMT-7) Timezone") | <kbd>**Admin**</kbd>, ```Maintainer``` | Prod / Issues / Biz |
-
-<BR>
-
-> [!Important]
-> When creating an ISSUE or a PR. Please TAG one (or more) of the Maintainers.
-
-Our Special Interest Groups (SIGs) are:
-| Functional area | Name | @GitHub | Area of Expertise | Product specialization |
-| :--- | :--- | :--- | :--- | :--- |
-| ``` Entire Product ``` | Tiago Castro | @tiagolobocastro | Architect / Eng | Mayastor |
-| ``` Data Plane ``` | Dmitry Savitskiy | @dsavitskiy | Architect / Eng | Mayastor |
-| ``` e2e Testing ``` | Blaise Dias | @blaisedias | Eng / Test / QA | All products | 
-| ``` e2e Testing ``` | Chris Denyer | @chriswldenyer | Eng / QA | Mayastor |
- 
----
-
-> [!Important]
-> FAQ's on SIG's, Contributors and Maintainers 
-<BR>
-
-## What is a _Special Interest Group (SIG)_?
-- SIGs are small teams working together on a specific area/topic/zone.
-- They may change at any time, and have no strict definition.
-- SIGs may be created, empowered, and destroyed by the maintainers at any time.
-
-
-## May I join a SIG?
-- Of course, we'd love that!
-
-
-## May I Become a maintainer?
-- Yes, we have a set of guidelines and rules explaining how to advance to Maintainer-ship...
-    - Governance rules, code of conduct, contributing and a ``` Maintainer Ladder ```  that details pre-reqs, criteria and responsibilities you need to agree to & meet
-    - Contribution history is critical
-    - PR review input, ISSUE management
-    - Attendance at community meetings are also important
-    - Feature contribution
-    - Active support engagement on our SLACK Channel
-    - Bug resolution
+  - All PR's are reviewed by the Maintainers and Eng/Dev first, before being approved and committed for Eng/development.
 
+This project follows the [OpenEBS Contributor Guidelines](https://github.com/openebs/community/blob/HEAD/CONTRIBUTING.md).
```

Comment:
Isn't this present on line 7 already?

Reply:
Oops, it was supposed to be different link. Fixed now.